locale/en_US/default.po

```diff
@@ -140,5 +140,7 @@ msgstr "R"
 msgid "default.contextSettings.emailSignature"
```

Comment:
This change would need to be flagged in all the other translations -- use the "fuzzy" tag to indicate that it should be reviewed for retranslation.

Reply:
Flagged (still at the OJS level, as soon as the locale value is ok I will move that to pkp-lib)